src/components/NavigationBar/NavigationBar.test.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { NavigationBar } from "./NavigationBar";
+import { create } from "react-test-renderer";
+import { shallow } from "enzyme";
+
+describe("<NavigationBar/>", () => {
+  it("renders the snapshot correctly", () => {
+    const component = create(
+      <NavigationBar
+        targets={[
+          {
+            name: "home",
+            url: "/",
+          },
+        ]}
+      />
+    );
+
+    expect(component).toMatchInlineSnapshot(`
+      <div
+        className="navigationBar-container"
+      >
+        <ul>
+          <nav>
+            <li
+              className="text-piped"
+              data-test="navTarget-name"
+            >
+              <a
+                className="govuk-link lbh-link"
+                href="//"
```

Comment:
This doesn't look right? Is it adding an extra forward slash onto your URLs?

Reply:
Have fixed this, there was an extra / being added to the slug variable within the href attribute on <Link/>